unime/src/i18n/en/index.ts

```diff
@@ -216,6 +222,30 @@ By using our Service, you acknowledge that you have read, understood, and agree
         TITLE_2: 'appearance',
         SUBTITLE: 'Are you more of a night owl?',
       },
+      SECURITY: {
+        LABEL: 'Security',
+        NAVBAR_TITLE: 'Security',
+        SWITCH_LABEL: 'Unlock with {type:string}',
+        BIOMETRIC_TYPE: {
+          ANDROID: {
+            FACE_ID: 'facial recognition',
+            TOUCH_ID: 'fingerprint',
+          },
+          IOS: {
+            FACE_ID: 'Face ID',
+            TOUCH_ID: 'Touch ID',
+          },
+          GENERIC: 'biometrics',
+        },
+        ENABLE: {
+          DIALOG_TITLE: 'Enable {type:string}',
```

Comment:
Wouldn't `Enable login with {type:string}` be slightly better?

Reply:
That's too long ... :/